### PR TITLE
Feature / create & style header

### PR DESCRIPTION
**Pull request by liemgumball**
- Added head logo
image

- Created and styled header in Mobile-first
image

- Applied hover animation to drop menu (which have authencation buttons)
image

- Styled for Tablet (the menu doesn't have authencation buttons)
image

- Styled for Desktop
image

### CODE REVIEW

**Comment by huyenpham-agilityio** [comment]
the menu on mobile is not good. Please refer other website

**Comment by liemgumball** [comment]
- Updated style the menu on mobile (tablet also)
image
image

**Comment by liemgumball** [comment]
- Updated style the menu on mobile (tablet also)
image
image

File: practice-2/src/assets/css/layout/header.css [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+.navbar {
+    grid-area: navbar;
+    position: relative;
+    align-items: center;
+}
+
+.navbar__menu-btn {
+    grid-area: menu-btn;
+    border: none !important;
```
Comment (huyenpham-agilityio) [inline practice-2/src/assets/css/layout/header.css:16]: avoid use important please, double check for other cases

File: practice-2/src/assets/css/layout/header.css [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+.navbar__menu {
+    display: none;
+    transition: opacity 1s ease;
+    row-gap: 5px;
+}
+
+.navbar__menu__auth-button {
+    width: 100%;
+    height: 30px;
+    border: 1px solid #eeeeee;
```
Comment (huyenpham-agilityio) [inline practice-2/src/assets/css/layout/header.css:33]: do you have variable for this color?

File: practice-2/src/assets/css/layout/header.css [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+    .navbar:hover .navbar__menu {
+        background-color: var(--background-color);
+        position: absolute;
+        display: flex;
+        flex-direction: column;
+        align-items: stretch;
+        right: 0;
+        width: 90vw;
+        padding: 1rem;
+        border: 1px solid #eeeeee;
```
Comment (huyenpham-agilityio) [inline practice-2/src/assets/css/layout/header.css:77]: ditto

File: practice-2/src/assets/css/layout/header.css [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+        box-shadow: 0 1px 3px rgba(31, 35, 40, 0.12),
```
Comment (huyenpham-agilityio) [inline practice-2/src/assets/css/layout/header.css:78]: this is duplicated somewhere, can you create variable?

**Review by huyenpham-agilityio — CHANGES_REQUESTED** [review]

**Review by huyenpham-agilityio — APPROVED** [review]

**Pull request merged**